test-server.js

```diff
@@ -27,12 +30,25 @@ describe('test server', function () {
             email: 'jimmy@gmail.com',
             password: 'iamnotfondoficecream1234',
         };
-        request(server).get('/api/get-user/2').expect(200).end((err, res) => {
-            expect(res.body.user.firstName).to.equal(user.firstName);
-            expect(res.body.user.email).to.equal(user.email);
-            expect(res.body.user.password).to.equal(user.password);
-            done();
-        });
+        request(server)
+            .get('/api/get-user/2')
+            .expect(200)
+            .end((err, res) => {
+                expect(res.body.user.firstName).to.equal(user.firstName);
+                expect(res.body.user.email).to.equal(user.email);
+                expect(res.body.user.password).to.equal(user.password);
+                done();
+            });
+    });
+
+    it('does not return user if user does not exist', (done) => {
+        request(server)
+            .get('/api/get-user/5')
+            .expect(200)
```

Comment:
```suggestion
            .expect(404)
```

should be a 404 in this case